web/src/apps/shoelace.ts

```diff
@@ -12,4 +13,9 @@ import { version } from '../../package.json'
 const { NODE_ENV } = process.env
 const base = NODE_ENV === 'development' ? 'dev' : version
 
-setBasePath(`~static/${base}/shoelace-style/`)
+setBasePath(`/~static/${base}/shoelace-style/`)
```

Comment:
OK, this maybe explains why we have to put the Shoelace CSS files in the `<head>`?

Reply:
I agree, it would be good to have rust ignore this. 

That's ☝️ partially the issue here. I haven't found a solution here I like - importing multiple css files into main causes some naming clash in parcel which other users had issues with as well. I would like to look at bundling `main.css` + other stylesheets into one ... I'd like to make a separate issue for this - it's not a show stopper right now, but should be addressed.